test.js

```diff
@@ -19,6 +19,7 @@ describe('updateNotifier', () => {
 			callback: options.callback || null
 		};
 	};
+	const argv = process.argv.slice(0);
```

Comment:
I think we actually need to move this logic within the `beforeEach` closure; otherwise line 32 only preserves the original array _after the first test_, meaning that line 57 is modifying the shallow copy and the original array is lost, which affects subsequent tests (even though it doesn't cause the tests to fail, it _does_ affect code coverage).

I suggest removing line 22 and making the subsequent lines look like this:

```js
	let argv;
	let configstorePath;

	beforeEach(() => {
		argv = process.argv.slice();
		configstorePath = updateNotifier(generateSettings()).config.path;
	});
```